### PR TITLE
Allow plugins modify request data for webhook campaign action

### PR DESCRIPTION
**Pull request by kuzmany**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 

#### Description:
This PR added dispatcher to request of webhook campaign action. 
Allow modify url, headers or payload by plugin.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Code review should be enought
3. Or ask me for plugin and instructions

### CODE REVIEW

File: app/bundles/WebhookBundle/Event/WebhookRequestEvent.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<?php
+
```
Comment (escopecz) [inline app/bundles/WebhookBundle/Event/WebhookRequestEvent.php:2]:
```suggestion

declare(strict_types = 1);

```

**Review by escopecz — DISMISSED** [review]
Looks good. I'd add the script types declaration to all new classes.

**Comment by npracht** [comment]
@kuzmany can you answer @escopecz feedback ? Also I'm wondering if a piece of doc would be helpful ? What do you think ?

File: app/bundles/WebhookBundle/Event/WebhookRequestEvent.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+declare(strict_types = 1);
```
Comment (escopecz) [inline app/bundles/WebhookBundle/Event/WebhookRequestEvent.php]:
```suggestion
declare(strict_types=1);
```
Oops, sorry Zdeno, I introduced a CS issue. This fixes it.

**Review by escopecz — APPROVED** [review]
Code looks good 👍

**Review by hluchas — APPROVED** [review]

**Pull request merged**